README.md

```diff
@@ -1,33 +1,66 @@
-# packer-st2
-Packer templates for building Vagrant box with [StackStorm](https://github.com/stackstorm/st2) community installed
+# StackStorm Vagrant & OVA
+[![Latest Release](https://img.shields.io/github/release/StackStorm/packer-st2/all.svg)](https://github.com/StackStorm/packer-st2/releases)
+[![Download from Vagrant Cloud](https://img.shields.io/badge/Vagrant-cloud%20%E2%86%92-1563ff.svg)](https://app.vagrantup.com/stackstorm/boxes/st2/)
 
-## Building Vagrant & OVA
+[Packer](https://www.packer.io/intro/index.html) templates with [InSpec](https://www.inspec.io/) integration tests for building Vagrant box & OVA image with [StackStorm](https://github.com/stackstorm/st2) community installed.
+A fully tested and packaged artifacts are produced during the build pipeline.
```

Comment:
I think you mean to say "A fully tested and packaged artifact is produced during the build pipeline." ?

Reply:
If there's more than one artifact, we should use "Fully tested and packaged artifacts are produced during the build pipeline."

Reply:
Thanks!
Will take the second one, there are indeed 2 artifacts: `.box` and `.ova`.